chain-signatures/node/src/protocol/presignature.rs

```diff
@@ -516,3 +518,15 @@ impl PresignatureManager {
         messages
     }
 }
+
+pub fn hash_as_id(triple0: TripleId, triple1: TripleId) -> PresignatureId {
+    use sha3::{Digest, Sha3_256};
+    let mut hasher = Sha3_256::new();
+    hasher.update(triple0.to_le_bytes());
+    hasher.update(triple1.to_le_bytes());
+    let seed: [u8; 32] = hasher.finalize().into();
+    let mut rng = StdRng::from_seed(seed);
+    let id = rng.gen::<u64>();
```

Comment:
Why not just pick first 8 byte of the `[u8; 32]` as u64 id?

Reply:
yeah that also works. Either could work, but I guess that one doesn't rely on RNG. Will change it to not use RNG